tensorflow/lite/micro/kernels/reduce.cc

```diff
@@ -29,6 +29,22 @@ namespace reduce {
 
 constexpr int kMaxNumberOfAxis = 4;
 constexpr int kMaxNumberOfReducedAxis = 2;
+enum ReduceType {
+  kMax,
+};
+
+struct OpData {
+  int32_t multiplier;
+  int shift;
+  int temp_buffer_idx;
+  int resolved_axis_idx;
+};
+
+void* InitMax(TfLiteContext* context, const char* buffer, size_t length) {
+  void* raw;
+  context->AllocatePersistentBuffer(context, sizeof(OpData), &raw);
+  return raw;
```

Comment:
Can you change to this (for consistency with the other kernels:
```
 TFLITE_DCHECK(context->AllocatePersistentBuffer != nullptr);
 return context->AllocatePersistentBuffer(context, sizeof(OpData));
```

And remove the unused params (buffer and length).

Reply:
fixed